src/main.scss

```diff
@@ -58,8 +58,9 @@ body, html {
   height: 100%;
   resize: none;
 
-  padding: 15px;
-  padding: 8px;
+  // Height of the search dialog. 
+  // If it's too big, subtract 4px (padding-top of .CodeMirror-lines) or remove entirely
+  padding-top: 21.08px;
```

Comment:
I think we should be good without this line

Reply:
So it's OK / preferred to have no padding on top?

Reply:
Yup, having an empty space on top when editing feels like something should be there. The demo doesn't have it, so it's fine

Reply:
OK, done in https://github.com/standardnotes/code-editor/pull/25/commits/314be5e81eb4a6c00c3b69b45b4d2eb9defbbda9